pipeline/scripts/AddOnFunctions/getArea.R

```diff
@@ -9,7 +9,7 @@ getArea <- function(mu,resol,scale,sigma,int.factor){
   #   sigma=sigma1
   
   # avoid too big vector (cannot allocate vector of size ...)
-  if (mu>1000) return(0)
+  if (mu>1200) return(0)
```

Comment:
Is this an arbitrary number?

Reply:
this is something that earlier developers put there. It was one of the main reasons why it was impossible to set a higher mass range than 1000 m/z (was always 600 m/z).
the mass range is retrieved in the .raw files in one of the first scripts of this pipeline. However I left this hardcoded. I was also thinking about just removing the whole if-statement, but I leave it for now, because of the comment above (too big vector). I haven't seen any problems yet.